### PR TITLE
Added a news component

### PR DESCRIPTION
**Pull request by navanchauhan**
Rather than using the Guardian API, I changed it to the news API v2

**Pull Request Label:**
* [ ] Bug
* [ ] New feature
* [ ] Enhancement
* [x] New component
* [ ] Typo

**Pull Request Checklist:**
- [x] Have you followed the [guidelines for contributing](https://github.com/alexanderepstein/Bash-Snippets/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/alexanderepstein/Bash-Snippets/pulls) for the same fix or component?
- [ ] Have you ran the tests locally with `bats tests`?

-----

### CODE REVIEW

File: news/news [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    fi
+  fi
+}
+
+usage()
+{
+  cat <<EOF
+News
+Description: Brings you the latest headline from a tag
+Usage: stocks [flag] or stocks [tag]
```
Comment (vitalkanev) [inline news/news]: Well, it should be `news` rather than `stocks`

Comment (navanchauhan) [reply news/news]: Oh, I forgot to change that. I was using stocks as a skeleton for this

**Comment by alexanderepstein** [comment]
Would be nice if this returned 5 headlines and then the user could choose which one to get more information about similar to the ytview script. Not sure if this is also possible but including the website that leads to the full article  in the final output would also be nice.

**Comment by navanchauhan** [comment]
Can’t to without jq

**Comment by alexanderepstein** [comment]
Any chance you will be including the top 3-5 headlines (without jq) similar to what you did for the pwned tool

**Comment by navanchauhan** [comment]
@alexanderepstein Forgot to update it, I'll try to do it within 48 hours

**Comment by navanchauhan** [comment]
I don't think multiple articles will happen

**Comment by alexanderepstein** [comment]
I will see what I can do, I think I can get multiple articles by utilizing a for loop when parsing the json response

**Comment by navanchauhan** [comment]
Is the following output ok ?
Command =: news Donald Trump
Output:
=============================================
| Title: Donald Trump Persists In Undermining Donald Trump
| Author: Thornton McEnery
| Link: https://dealbreaker.com/2017/10/donald-trump-persists-in-undermining-donald-trump/
| Description: He's the Plaxico Burress of policy.
=============================================

**Comment by navanchauhan** [comment]
screenshot from 2017-12-16 12-57-54

**Comment by navanchauhan** [comment]
Is the output fine ?

**Comment by alexanderepstein** [comment]
The output itself is fine but I think it is worth trying to grab the top 3

**Comment by alexanderepstein** [comment]
Can you point me to the file in your fork it seems github has lost it, I cant work on it

**Comment by navanchauhan** [comment]
It's not possible to get three different articles in the same format, I've tried different codes but they don't work. I think we should close this PR for the time being ?

**Comment by alexanderepstein** [comment]
I will close this for now

**Comment by navanchauhan** [comment]
[news.txt](https://github.com/alexanderepstein/Bash-Snippets/files/1606194/news.txt)
Just replace txt with sh and this would be good to go

**Pull request closed without merging**